proarc-common/src/main/java/cz/cas/lib/proarc/common/object/emods/BdmArticleMapper.java

```diff
@@ -38,6 +42,66 @@
     public static final String GENRE_PEER_REVIEWED_TYPE = "peer-reviewed";
 
     @Override
+    public void createMods(ModsDefinition mods, Context ctx) {
+        super.createMods(mods, ctx);
+
+        List<PhysicalDescriptionDefinition> listPhysicalDescription = new ArrayList<>();
+        for (PhysicalDescriptionDefinition pd : mods.getPhysicalDescription()) {
+            for (FormDefinition form : pd.getForm()) {
+                FormDefinition newFormDefinition = new FormDefinition();
+                if ("bez média".equals(form.getValue())) {
+                    setFormDefinition(form, newFormDefinition, "svazek");
+                } else if ("počítač".equals(form.getValue())) {
+                    setFormDefinition(form, newFormDefinition, "online zdroj");
+                } else if ("jiný".equals(form.getValue()) && (form.getAuthority() == null || "rdamedia".equals(form.getAuthority()))) {
+                    setFormDefinition(form, newFormDefinition, "jiný");
+                }
```

Comment:
Možná by stálo za to přidat `else` pro případ neznámé hodnoty, aby jen nedošlo k jejímu zahození - nebo zahození hodnoty někde poznamenat do logu, pokud je zahození požadované.

Reply:
V případě neznámé hodnoty se hodnota nezahazuje, nechá se, jen se k ní nevygeneruje nový element. Proto zde není řešen navrhovaný `else`.

Reply:
Ok, v tom případě toto za mě může být.